packages/epics/src/people/components/people-transfer-form.tsx

```diff
@@ -58,6 +62,7 @@ export const PeopleTransferForm = ({
           token: undefined,
         },
       ],
+      memo: '',
```

Comment:
Default value should be null or undefined as we don't want to pollute the DB with empty memos
```suggestion
      memo: null,
```